### ISSUE TITLE
Cannot create notebook in Brave

### ISSUE THREAD

**Issue by rachids** [phpsandbox/phpsandbox#3]
### **Description of Bug**

I tried to test PHPSandbox from Brave browser but it's getting stuck at `Connecting to Notebook` screen.
It works very quickly in Firefox though.

I tried with and without the Brave's protection (blocking tracking js and other stuff)

### **How can we reproduce it?**

1. Create standard notebook with Brave
2. It seem to be stuck

### **Expected results**
Notebook should be created and connected to.

### **Actual Behavior**
It seems a websocket connection is closed.
I also have some 401 to this endpoint `GET https://phpsandbox.io/api/user 401`

### **Screenshots or Screencast**

**My console log with protection activated**:
![image](https://user-images.githubusercontent.com/4490551/126007397-707f1949-011a-4dea-bf13-8d92e45b2690.png)

**My console log without protection activated**:
![image](https://user-images.githubusercontent.com/4490551/126007441-fdcd3e89-1df1-4ef3-9018-2fca000ab3eb.png)

**Comment by mercyikpe** [comment]
Hi @rachids thanks for reporting this issue. Are you still experiencing this?

I tried recreating this issue on brave. I was able to create a Standard notebook.

Here is a screenshot with shield protection on.
![image](https://user-images.githubusercontent.com/29842348/126508474-aa4c41ff-56c6-4cd0-84f9-cd278b4a2117.png)


With shield protection off.
![image](https://user-images.githubusercontent.com/29842348/126508609-1f98b155-40c2-430e-83b9-572ba6d64a39.png)
